core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMetadata.java

```diff
@@ -887,6 +887,7 @@ default Optional<ConnectorOutputMetadata> finishRefreshMaterializedView(
             ConnectorInsertTableHandle insertHandle,
             Collection<Slice> fragments,
             Collection<ComputedStatistics> computedStatistics,
+            List<String> sourcePolymorphicTableFunctions,
```

Comment:
any reason to have this before `List<ConnectorTableHandle> sourceTableHandles`?
i would add this new param as the last one

Reply:
same in metadata

Reply:
I just didn't want to add a parameter as the last to avoid conflicts :|